### ISSUE TITLE
Headless mode should fail if CT/CS statements are missing column defs

### ISSUE THREAD

**Issue by big-andy-coates** [confluentinc/ksql#2205]
Requested by @hjafarpour 

Headless mode should _not_ support getting schema from the schema registry at the moment.

Interactive mode gets the schema _once_ and bakes this into the command in the command topic.

We can look at support schema evolution for both headless and interactive later. But for now, with not testing around evolution, we should not allow it in headless mode.

**Comment by miguno** [comment]
Related: https://github.com/confluentinc/ksql/issues/1530

**Comment by big-andy-coates** [comment]
Decision made not to implement this, (as it may break existing customers who are carefully managing the evolution of their schemas).  The true solution is to support schema inference in headless mode, i.e. #1530.
